meson.build

```diff
@@ -516,15 +523,22 @@ else
   py3 = pymod.find_installation()
 endif
 
+config_ratbagctl = configuration_data()
+config_ratbagctl.set('RATBAGD_API_VERSION', ratbagd_api_version)
+config_ratbagctl.set('version', meson.project_version())
+
+ratbagctl_in = configure_file(input : 'tools/ratbagctl.in.in',
```

Comment:
Hmm... that's a lot of `.in`

Reply:
yeah, but `ratbagctl.in.out` was more confusing :package: 

using `.in.in` is a standard pattern afaik for doubly-processed files.